docs/next.config.js

```diff
@@ -19,6 +19,15 @@ const isDeployPreview = Boolean(process.env.PULL_REQUEST_ID);
 const buildOnlyEnglishLocale = isDeployPreview && !l10nPRInNetlify && !vercelDeploy;
 
 module.exports = withDocsInfra({
+  async redirects() {
+    return [
+      {
+        source: '/',
+        destination: '/material-ui/getting-started/overview/',
+        permanent: true,
```

Comment:
```suggestion
```

This just distinguishes between a 301 and a 302. A 301 is permanent and therefore remembered by caches / bots / etc. I'd probably err towards just using a 302 since this doesn't really matter in our case and makes it easier to change.